db/migrate/201603021905_convert_folder_areas_to_enums.rb

```diff
@@ -0,0 +1,58 @@
+class ConvertFolderAreasToEnums < ActiveRecord::Migration
+
+  class OldFolder < ActiveRecord::Base
+    AREAS = ['ESS', 'MMWW', 'SWT', 'MUA', 'Grundst. Info', 'Mathe', 'Sonstiges']
+    enum area: [:sonstiges, :ess, :mmww, :swt, :mua, :grundstudium, :mathe]
+  end
+
+
+  def up
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Cyclomatic complexity for up is too high. [7/6]](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2173675780/issues/source?bid=3141297&fileBranchId=3152185#l9)